gravitee-apim-portal-webui-next/src/entities/subscription/subscription.fixture.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2024 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { isFunction } from 'rxjs/internal/util/isFunction';
+
+import { Subscription } from './subscription';
+
+export function fakeSubscriptionResponse(
+  modifier?: Partial<Subscription> | ((baseSubscription: Subscription) => Subscription),
+): Subscription {
+  const base: Subscription = {
+    data: [
+      {
```

Comment:
Might as well create the subscription.fixture.ts as well and include in the data here: `[fakeSubscription()]`

Reply:
Done